js/state_modifiers.js

```diff
@@ -64,9 +68,15 @@ export const unified_source_modifier = {
         return acc + curBlock._pagesList.length
       }
     }
-    return window.book.upload_blocks
-      .filter(x => x.pdfDoc != null)
-      .reduce(lookForPage, 0)
+    if (is_interlaced) {
+      const block = (pageNum % 2 == 0) ? valid_upload_blocks[0] : valid_upload_blocks[1]
+      const block_i = window.book.upload_blocks.findIndex(b => b == block)
+      const pageListEntry = block._pagesList[pageNum]
+      return (pageListEntry == -1 || pageListEntry == undefined)
```

Comment:
this chunk o code duplicates some stuff up above  - should probably be pulled out into a wee helper function but... not right now